.github/workflows/config/spelling_allowlist.txt

```diff
@@ -1,4 +1,5 @@
 ABI
+AFQMC
```

Comment:
Most of these words (with the possible exception of this one) should not be added to the allow list. Rather, they need to marked as source code artifacts by annotating them with backticks in the original file that is producing this error. The spell checker will ignore items that are annotated with backticks.